src/main.py

```diff
@@ -94,6 +98,7 @@
 #  there as well.
 
 _CONFIG = {
+    'template': base.constants.JINJA2,
```

Comment:
can we set base.constants.CLOSURE as default? If you don't the examples/xss demo throws an exception.